Package.swift

```diff
@@ -460,7 +460,7 @@ let package = Package(
         .package(url: "https://github.com/SwiftGen/StencilSwiftKit", exact: "2.10.1"),
         .package(url: "https://github.com/SwiftGen/SwiftGen", exact: "6.6.2"),
         .package(url: "https://github.com/tuist/XcodeProj", exact: "8.19.0"),
-        .package(url: "https://github.com/cpisciotta/xcbeautify", from: "2.4.0"),
+        .package(url: "https://github.com/cpisciotta/xcbeautify", branch: "main"),
```

Comment:
@cpisciotta do you have a release planned including this change?

Reply:
I don't understand your question. `xcbeautify` should always be used via a semantic version tag. `main` is always "release-ready," but often doesn't include changes that should be bundled with a release. I recommend reverting this change.

Why are switching from tag to branch?

---

Separately, `2.4.1` is the latest version.

Reply:
[This change](https://github.com/cpisciotta/xcbeautify/commit/689eb1b97806af969086e80e146a42d98956920b) is unreleased. It might have gone unnoticed because it was automated. So I was wondering if you'll prepare a 2.5.0 release including it.

Reply:
Thanks Charles :). I updated to 2.5.0 and the dependency resolution ran smoothly.